recipe/meta.yaml

```diff
@@ -26,29 +26,37 @@ requirements:
     - python
 
 test:
+  source_files:
+    - tests
   requires:
     - pip
     - pytest
     - pytest-cov
   commands:
     - pip check
+    - pytest tests/test_pegen.py -vv
   imports:
     - pegen
+    - pegen.grammar
+    - pegen.grammar_parser
+    - pegen.grammar_visualizer
+    - pegen.parser
+    - pegen.parser_generator
+    - pegen.python_generator
+    - pegen.tokenizer
+    - pegen.utils
 
 about:
   home: https://we-like-parsers.github.io/pegen/
```

Comment:
Strictly speaking, the homepage switched to the GH repo, https://github.com/we-like-parsers/pegen/blob/main/pyproject.toml#L39.  I don't know how much it matters.